quasar.conf.js

```diff
@@ -14,8 +14,8 @@ module.exports = function (ctx) {
             // 'axios',
             // 'google-api',
             // 'jwt',
-            // 'vue-analytics',
             // 'vuelidate',
+            // 'vue-gtm'
```

Comment:
Let's have this always enabled?

Reply:
https://github.com/anyTV/quasar-boilerplate/pull/32/commits/0cacf6a1b4aab80396a4b5b0cc4a3fde11929c70